tests/test_organization.py

```diff
@@ -179,3 +179,47 @@ def test_create_relationship(setUp,
 
     fake_signal.assert_called_with(object_id=composition.id,
                                    sender=Composition)
+
+def is_tab_active(page, tab_name):
+    if 'primary">{}'.format(tab_name) in page.content.decode('utf-8'):
+        return True
+    else:
+        return False
+
+
+@pytest.mark.django_db
+def test_organization_edit_buttons(setUp,
+                                   full_organizations,
+                                   membership_person,
+                                   membership_organization):
+     org = full_organizations[0]
+     composition = Composition.objects.first()
+     person = org.personnel[0]
+     association = org.associations[0]
+     emplacement = org.emplacements[0]
+
+     assert is_tab_active(setUp.get(reverse_lazy('edit-organization', args=[org.uuid])),
+                          'Basics') == True
+
+     assert is_tab_active(setUp.get(reverse_lazy('create-organization-composition', args=[org.uuid])),
+                          'Relationships') == True
+     assert is_tab_active(setUp.get(reverse_lazy('edit-organization-composition', args=[org.uuid, composition.pk])),
+                          'Relationships') == True
+
+     assert is_tab_active(setUp.get(reverse_lazy('create-organization-personnel', args=[org.uuid])),
+                          'Personnel') == True
+     assert is_tab_active(setUp.get(reverse_lazy('edit-organization-personnel', args=[org.uuid, person.pk])),
+                          'Personnel') == True
+
+     assert is_tab_active(setUp.get(reverse_lazy('create-organization-emplacement', args=[org.uuid])),
+                          'Locations') == True
+     assert is_tab_active(setUp.get(reverse_lazy('edit-organization-emplacement', args=[org.uuid, emplacement.pk])),
+                          'Locations') == True
+
+     assert is_tab_active(setUp.get(reverse_lazy('create-organization-association', args=[org.uuid])),
+                          'Locations') == True
+     assert is_tab_active(setUp.get(reverse_lazy('edit-organization-association', args=[org.uuid, association.pk])),
+                          'Locations') == True
+
+     assert is_tab_active(setUp.get(reverse_lazy('create-organization-membership', args=[org.uuid])),
+                          'Relationships') == True
```

Comment:
I wasn't sure how to test `edit-organization-membership`. Every `pk` from `org` attributes I tried failed.

Reply:
I wasn't really sure what the right `pk` was either, so here's how I figured it out! First I checked `organization/urls.py` to see how the URL was structured:

https://github.com/security-force-monitor/sfm-cms/blob/c5a163aee02ab1db810d3760c2b9c7c431bcad35/organization/urls.py#L40-L42

Then I checked `OrganizationEditMembershipView` to see how `pk` was being used. Looks like it's the primary key for the `MembershipOrganization` model:

https://github.com/security-force-monitor/sfm-cms/blob/c5a163aee02ab1db810d3760c2b9c7c431bcad35/organization/views.py#L334-L339

Our test includes the `membership_organization` fixture, so we should have access to it:

https://github.com/security-force-monitor/sfm-cms/blob/93e255df7252a5f2783e9308323fd2311b2c99ac/tests/test_organization.py#L190-L194

Unfortunately, the `membership_organization` fixture currently doesn't return anything. I had to make a quick adjustment to it to give us access to the `MembershipOrganization` object:

```diff
--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -399,7 +399,7 @@ def membership_organization(organizations, access_points):
         },
     }

-    MembershipOrganization.create(mem_info)
+    return MembershipOrganization.create(mem_info)
```

After that, passing in `membership_organization,pk` as an `arg` to `reverse_lazy` worked!